src/app/api/auth/next-auth.ts

```diff
@@ -33,16 +43,27 @@ const nextAuth = NextAuth({
     },
   },
   providers: ENABLE_OAUTH_SSO
-    ? [
-        Auth0({
-          // Specify auth scope, at least include 'openid email'
-          // all scopes in Auth0 ref: https://auth0.com/docs/get-started/apis/scopes/openid-connect-scopes#standard-claims
-          authorization: { params: { scope: 'openid email profile' } },
```

Comment:
Hi, please keep this line and the annotation above, because it is an example of declaring OAuth authorization scopes, which is the information we need to display at frontend.

Reply:
Hi cy948,
Had add those code back, https://github.com/lobehub/lobe-chat/pull/1456/commits/b468089bb7978a258767df30820a79bea2125d66